packages/patternfly-4/react-core/src/components/Modal/Modal.js

```diff
@@ -23,19 +23,25 @@ const propTypes = {
   actions: PropTypes.any,
   /** A callback for when the close button is clicked */
   onClose: PropTypes.func,
+  /** Default width of the Modal. */
```

Comment:
Should we add clarification, something like `Use either the width or the isLarge/isSmall prop`

Reply:
I will check this out.
Although if I am not wrong `isLarge` and `isSmall` are setting `maxWidth` and `minWidth` to `Modal` respectively.

Thanks.